src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -68,59 +81,47 @@ class HaConfigHardware extends LitElement {
               <div class="content">
                 <ha-card outlined>
                   <div class="card-content">
-                    <ha-settings-row>
-                      <span slot="heading"
-                        >${this.hass.localize(
-                          "ui.panel.config.hardware.board"
-                        )}</span
-                      >
-                      <div slot="description">
-                        <span class="value">${this._OSData.board}</span>
-                      </div>
-                    </ha-settings-row>
+                    ${this._OSData?.board
+                      ? html`
+                          <ha-settings-row>
+                            <span slot="heading"
+                              >${BOARD_NAMES[this._OSData.board] ||
+                              this.hass.localize(
+                                "ui.panel.config.hardware.board"
+                              )}</span
+                            >
+                            <div slot="description">
+                              <span class="value">${this._OSData.board}</span>
+                            </div>
```

Comment:
I don't think that it's necessary to show the raw value.

Reply:
I think it adds some secondary text. Similar to Matthias's mockup. Otherwise it feels empty